pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -171,6 +171,7 @@ Panels::CpuPanelEE::CpuPanelEE( wxWindow* parent )
 	*this += m_button_RestoreDefaults | StdButton();
 
 	Connect( wxID_DEFAULT, wxEVT_COMMAND_BUTTON_CLICKED, wxCommandEventHandler( CpuPanelEE::OnRestoreDefaults ) );
+	Connect( wxID_ANY, wxEVT_COMMAND_RADIOBUTTON_SELECTED, wxCommandEventHandler(CpuPanelEE::EECache_Event) );
```

Comment:
Thanks for the suggestion. Initially I was planning on using ``Bind()`` but since the code was filled with ``Connect()`` I decided against it and used ``Connect()`` instead to have the code consistent.

Also I thought potentially it could cause some.. _hmm_  conflicts with WX 2.8 (or) something and didn't really bother with it.

since you're fine with it - I have changed it to bind instead :)